docs/tutorials/noise.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>exactly the same with the above. this could be a helper function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/quantum/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>